src/templates/React InstantSearch widget/package.json.hbs

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "{{ packageName }}",
+  "version": "1.0.0",
+  "description": "{{ description }}",
+  "keywords": [
+    "instantsearch-widget",
+    "instantsearch",
+    "react-instantsearch",
```

Comment:
I think we need this so that we can filter widget repos by flavor. We don't have 'instantsearch.js' for the vanilla widget template. We should add it there too. What do you think?

Reply:
yes, we should add it there too, although IIRC we agreed on `(react-)Instantsearch-widget-xxx` (not yet updated in the vanilla template), so filtering can be done with name as well

Reply:
done [`f1f5105` (#456)](https://github.com/algolia/create-instantsearch-app/pull/456/commits/f1f5105411c336f424af898077dbb4c7abc0315d)